intel-lustre-client-server/azuredeploy.json

```diff
@@ -0,0 +1,1222 @@
+					},
+					"subnet": {
+						"id": "[variables('subnetServersID')]"
+					}
+				}
+			}]
+		}
+	}, {
+		"apiVersion": "2015-06-15",
+		"type": "Microsoft.Compute/virtualMachines",
+		"name": "[concat(parameters('vmNamePrefix'),'mgs')]",
+		"plan": {
+			"publisher": "intel",
+			"product": "lustre-cloud-edition-cv-image",
+			"name": "eval-lustre-2-7"
+		},
+		"location": "[parameters('location')]",
+		"dependsOn": [
+			"storageAccountsCopy",
+			"[concat('Microsoft.Network/networkInterfaces/', parameters('vmNamePrefix'), 'mgs')]"
+		],
+		"tags": {
+			"LustreType": "mgt"
+		},
+		"properties": {
+			"hardwareProfile": {
+				"vmSize": "[parameters('mgsVmSize')]"
+			},
+			"osProfile": {
+				"computerName": "[concat(parameters('vmNamePrefix'),'mgs')]",
+				"adminUserName": "[parameters('adminUsername')]",
+				"adminPassword": "[parameters('adminPassword')]",
+				"linuxConfiguration": "[variables('linuxConfiguration')]"
+			},
+			"storageProfile": {
+				"imageReference": {
+					"publisher": "[variables('imageReferenceLustre').publisher]",
+					"offer": "[variables('imageReferenceLustre').offer]",
+					"sku": "[variables('imageReferenceLustre').sku]",
+					"version": "latest"
+				},
+				"osDisk": {
+					"name": "osdisk",
+					"vhd": {
+						"uri": "[concat('http://', variables('storageAccountName'), variables('mgsStorageAccountNumber'), '.blob.core.windows.net/vhds/', concat(parameters('vmNamePrefix'),'mgs'), '-osdisk.vhd')]"
+					},
+					"caching": "ReadWrite",
+					"createOption": "FromImage"
+				},
+				"dataDisks": [{
+					"name": "datadisk0",
+					"diskSizeGB": 5,
+					"lun": 0,
+					"vhd": {
+						"Uri": "[concat('http://', variables('storageAccountName'), variables('mgsStorageAccountNumber'), '.blob.core.windows.net/vhds/', concat(parameters('vmNamePrefix'),'mgs'), '-datadisk0.vhd')]"
+					},
+					"caching": "None",
+					"createOption": "Empty"
+				}, {
+					"name": "datadisk1",
+					"diskSizeGB": 16,
+					"lun": 1,
+					"vhd": {
+						"Uri": "[concat('http://', variables('storageAccountName'), variables('mgsStorageAccountNumber'), '.blob.core.windows.net/vhds/', concat(parameters('vmNamePrefix'),'mgs'), '-datadisk1.vhd')]"
+					},
+					"caching": "None",
+					"createOption": "Empty"
+				}]
+			},
+			"networkProfile": {
+				"networkInterfaces": [{
+					"id": "[resourceId('Microsoft.Network/networkInterfaces',concat(parameters('vmNamePrefix'),'mgs'))]"
+				}]
+			}
+		}
+	}, {
+		"apiVersion": "2015-06-15",
+		"type": "Microsoft.Compute/virtualMachines/extensions",
+		"name": "[concat(parameters('vmNamePrefix'), 'mgs', '/init')]",
+		"location": "[parameters('location')]",
+		"dependsOn": [
+			"[concat('Microsoft.Compute/virtualMachines/', concat(parameters('vmNamePrefix'), 'mgs'))]"
+		],
+		"properties": {
+			"publisher": "Microsoft.OSTCExtensions",
+			"type": "CustomScriptForLinux",
+			"typeHandlerVersion": "1.3",
```

Comment:
Use 1.5 and add autoUpgradeMinorVersion property to all custom script extension definitions